python/lsst/ctrl/mpexec/separablePipelineExecutor.py

```diff
@@ -241,6 +239,7 @@ def run_pipeline(
         graph: lsst.pipe.base.QuantumGraph,
         fail_fast: bool = False,
         graph_executor: QuantumGraphExecutor | None = None,
+        num_proc: int = 1,
```

Comment:
While I'm fine with changing the default to 1, I'm less fine about encoding that default in a parameter. The point of `SeparablePipelineExecutor` is to let the user plug in customized components as needed. I'm now wondering how `fail_fast` got in there; it doesn't seem to be something requested on #230.